### DISCUSSION TITLE
Adapt the content in header according to language

### DISCUSSION THREAD

**Discussion by ntluong95** [quarto-dev/quarto-cli#9284] [Q&A] [answered]
### Description

I have this multilanguage book, thanks by babelquarto: https://epirhandbook-quarto.netlify.app/
As you may noticed, in the header, there is a banner "Need help learning R? Enroll in Applied Epi's [intro R course](https://www.appliedepi.org/live/), try our [free R tutorials](https://www.appliedepi.org/tutorial/), post in our [Community Q&A forum](https://community.appliedepi.org/), or ask about our [R Help Desk service](mailto:contact@appliedepi.org).", which is unchanged when the language version is changed. This was created by

```
format:
  html: 
    include-in-header:
      - google_tag.html
```
I want the banner to be displayed in different languages, based on the language version is chosen. Is there any way to do it?

**Comment by mcanouil** [comment, marked as answer]
Unfortunately, you used a third party to make your multilingual book and you use a custom file to make the banner.
Even if the two are built based on Quarto, those are independent of Quarto.

Note that Quarto 1.5 offers a built in way to make banner.

This being said, you need to add your "translation" for your banner and use JavaScript to change the content based on the language/link you activate (out of Quarto scope).

**Reply by ntluong95** [reply]
Hi Thanks for your reply. I was able to using JS to change the banner content based on language used
